src/lib/create/valid.js

```diff
@@ -4,7 +4,7 @@ import getParsingFlags from '../create/parsing-flags';
 import some from '../utils/some';
 
 export function isValid(m) {
-    if (m._isValid == null) {
+    if (m._isValid === null || m._isValid === true) {
```

Comment:
This changes validity evaluation from one to continual. I don't see why this change would need to cause that.